lib/Frontend/CompilerInvocation.cpp

```diff
@@ -2769,6 +2770,7 @@ static bool ParseSILArgs(SILOptions &Opts, ArgList &Args,
   Opts.DisableSILPartialApply |=
     Args.hasArg(OPT_disable_sil_partial_apply);
   Opts.VerifySILOwnership &= !Args.hasArg(OPT_disable_sil_ownership_verifier);
+  Opts.VerifySILOwnership |= CONDITIONAL_ASSERT_enabled();
```

Comment:
`VerifySILOwnership` is enabled anyway - unless explicitly disabled with the `-disable-sil-ownership-verifier` option.

Reply:
I removed this.